docs/cost/index.md

```diff
@@ -95,6 +95,12 @@ The following mapping tables describe:
 
 #### Implementation
 
+Disclosures in the implementation section of the CoST IDS relate to changes made after the contract award to the contract value, duration and scope.
+
+Where OCDS data is available the presence of changes may be detected by comparing values in the most recent OCDS release with a compiledRelease created from all prior releases.
```

Comment:
> If OCDS data is available, these changes may be determined by comparing the most recent OCDS release to a …

Should we refer to the more specific instructions in the mapping guidance? Would a reader know what a "compiledRelease created from all prior releases" means?

Reply:
The mapping refers to specific fields which should be monitored for changes, but doesn't describe how that monitoring can be performed.

This sentence gives guidance on how changes in OCDS data can be detected.

I've clarified the guidance in https://github.com/open-contracting/infrastructure/pull/103/commits/46ff5708c157c89fd5636616b8121b77c0caec08